internal/provider/resource_access_policy.go

```diff
@@ -24,9 +24,15 @@ func mapAccessPolicyToData(res *aidbox.AccessPolicy, data *schema.ResourceData)
 	data.SetId(res.ID)
 	data.Set("description", res.Description)
 	data.Set("engine", res.Engine.ToString())
-	if res.Link != "" {
-		data.Set("link", res.Link)
+	var linkData []interface{}
+	for _, ref := range res.Link {
+		data := map[string]interface{}{
```

Comment:
nit: are `ResourceId` and `ResourceType` both `string`? In which case the map type could be more specific
```suggestion
		data := map[string]string{
```

Reply:
Changed here https://github.com/patientsknowbest/terraform-provider-aidbox/pull/28/commits/c19cbb007bed2baba806ef296f44c0d60ab7fe70